x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/exporter/local/LocalExporterIntegTests.java

```diff
@@ -258,9 +257,9 @@ private void checkMonitoringDocs() {
         if (customTimeFormat == null) {
             customTimeFormat = "YYYY.MM.dd";
         }
-
-        DateTimeFormatter dateParser = ISODateTimeFormat.dateTime().withZoneUTC();
-        DateTimeFormatter dateFormatter = DateTimeFormat.forPattern(customTimeFormat).withZoneUTC();
+//TODO confirm this is the right format
```

Comment:
looks ok to me..
ISODateTimeFormat:
yyyy-MM-dd'T'HH:mm:ss.SSSZZ
when 
DateFormatters.STRICT_DATE_TIME
 (yyyy-MM-dd'T'HH:mm:ss.SSSZZ).